docs/static/skillmap/space/test.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
small: i don't think this file is necessary/being used. not blocking for this PR, feel free to delete in the next one when you're removing old files